src/main/java/cinema/entity/User.java

```diff
@@ -0,0 +1,20 @@
+package cinema.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class User {
+
+    Long id;
+    String login;
+    String password;
+    Role userRole;
```

Comment:
why is it not private? and others entities and dtos

Reply:
fixed